Svc/SeqDispatcher/SeqDispatcherEvents.fppi

```diff
@@ -0,0 +1,23 @@
+event INVALID_SEQUENCER(
```

Comment:
Typically in F´  core, we use UPPER_SNAKE_CASE command and parameter names for historical consistency with other missions.  For events and channels we use PascalCase.

Reply:
Okay, will make this change.